### PR TITLE
I removed wget in favor of curl.

### PR DESCRIPTION
**Pull request by tdensmore**
Often people don't have wget. wget is easy enough to install, but `curl` is more widely available by default. 

This commit will need some Windows testing love, since I don't have Windows to test with.

### CODE REVIEW

**Comment by AmberAlston** [comment]
@tdensmore @dexhorthy Are you still planning to push forward on this requested content change?

**Comment by tdensmore** [comment]
I would love to get this released. What do I need to do?

**Comment by AmberAlston** [comment]
@tdensmore  Since you requested @dexhorthy as a reviewer, push on him to help validate your suggested changes are useful improvements, and mark PR as reviewed. In the meantime you've got lots of quirky formatting (missing space between paragraphs, extra space between words and punctuation). Give it a once over on https://deploy-preview-343--kotsio.netlify.app/reference/v1beta1/application/ and fix the formatting issues. The current in-page headers in this pr are a big change in styling. If you intend to suggest that, please add @jgruica or @GraysonNull as a final reviewer post Dex review and your formatting fixes.

**Comment by tdensmore** [comment]
I have updated the formatting @AmberAlston . @dexhorthy can you look at this and approve if you like it?

**Comment by tdensmore** [comment]
Actually would either @cremerfc or @aj-jester be able to review these changes? They are minor: one functionality change (using curl instead of wget) and documentation changes to the kots.io website

**Comment by AmberAlston** [comment]
Bump on request for review @cremerfc @aj-jester @dexhorthy

**Comment by AmberAlston** [comment]
Kicking this PR again in case you want to move forward with it. If not, we're going to close it as stale in the near future.

**Comment by tdensmore** [comment]
Hey gang - I wanted to get some eyeballs on this before @AmberAlston marks it as stale. It is a minor change to remove `wget` in favor of `curl` ...

File: Makefile [hunk 1]
```diff
@@ -18,14 +18,14 @@ deps:
 	@echo '-> Downloading Hugo CLI to ./deps '; \
 	mkdir -p deps/; \
 	if [[ "$(distname)" == "Windows" ]]; then \
-	  wget --no-check-certificate -O hugo.zip $(url).zip; \
+	  curl -k $(url).zip -o hugo.zip; \
 	  unzip hugo.zip -d deps; \
 	else \
-	  wget -O hugo.tar.gz $(url).tar.gz; \
+	  echo $(url).tar.gz; \
```
Comment (jdewinne) [inline Makefile:24]: Why is the `echo` in here?

**Comment by jdewinne** [comment]
I would split the `wget` to `curl` and the documentation changes into 2 separate PR's. Or are they related?

File: Makefile (hunk 1 above)
Comment (tdensmore) [reply Makefile:24]: Good question. it was probably for debugging. I will remove it.

**Comment by tdensmore** [comment]
> I would split the `wget` to `curl` and the documentation changes into 2 separate PR's. Or are they related?

I will split these out... it has been so long that I forgot they were in there.

**Comment by tdensmore** [comment]
I will close this in favor of the two new PRs I created. One for `curl`, the other for updated documentation.

**Pull request closed without merging**